ajax/libs/emergence.js/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "emergence.js",
+  "filename": "emergence.min.js",
+  "description": "Detect element visibility in the browser",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/xtianmiller/emergence.js.git"
+  },
+  "homepage": "https://xtianmiller.github.io/emergence.js/",
+  "keywords": [
+    "elements",
+    "emergence",
+    "DOM"
```

Comment:
Yes, it's good to add more keywords, such as https://github.com/xtianmiller/emergence.js/blob/master/bower.json#L11-L16
Maybe we can as the same with upstream. 😃 
What do you think?

Thank you.

Reply:
Yeah! I think we could retain the same.

Reply:
@ParkourKarthik So would you modify this part to make this PR better to merge?
Thank you.

Reply:
Sure :blush: